### PR TITLE
fix: improve `Segment` button selected state style

### LINKED ISSUE #3300: Style of list/tree button group is confusing
I find this styling of the buttons confusing:

image

Which button is currently pressed? For me, it looks like the "tree" button (right) is pressed, which means that the clickable button is the "list" button (left), but this isn't the case.

I suggest switching the styles of the pressed and unpressed states.

If this sounds like a good idea, I can create a PR.

### PR DESCRIPTION
**Pull request by eyalch**
The changes here swap the styles of the _selected_ and _unselected_ states of the `Segment` button component.

Resolves #3300

### CODE REVIEW

**Comment by eyalch** [comment]
I haven't yet tested this locally, so if someone could help that'll be great!

**Comment by eyalch** [comment]
> I haven't yet tested this locally, so if someone could help that'll be great!

Tested locally. LGTM and behaves as I expect.

**Comment by PavelLaptev** [comment]
@eyalch thanks for the PR! I'll check it out 🙂

File: gitbutler-ui/src/lib/components/SegmentControl/Segment.svelte [hunk 1]
```diff
@@ -37,7 +37,7 @@
 	role="tab"
 	aria-selected={isSelected}
 	aria-disabled={disabled}
-	tabindex={isSelected ? 0 : -1}
+	tabindex={!isSelected ? 0 : -1}
```
Comment (eyalch) [inline gitbutler-ui/src/lib/components/SegmentControl/Segment.svelte]: @PavelLaptev Consider avoiding this double-negative.

```suggestion
	tabindex={isSelected ? -1 : 0}
```

Comment (PavelLaptev) [reply gitbutler-ui/src/lib/components/SegmentControl/Segment.svelte]: Thanks!

**Comment by PavelLaptev** [comment]
Looks good to me. I'll merge it in 10 mins 🙂 

https://github.com/gitbutlerapp/gitbutler/assets/18498712/7cf735dd-0e7d-4980-83e0-9da544afabfb

**Pull request merged**